test/lib/util/messenger.js

```diff
@@ -0,0 +1,61 @@
+require('dotenv').config({ silent: true })
+
+const axios = require('axios')
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+const Messenger = require('../../../lib/util/messenger')
+const sinon = require('sinon')
+
+chai.use(chaiAsPromised)
+chai.should()
+
+describe('Messenger', function() {
```

Comment:
I've never worked with Mocha/Chai before, but figure it was worth giving a shot. Not totally sure what I'm doing quite yet.